### DISCUSSION TITLE
C64c with 250466 (longboard) which SIDkick variant should I use?

### DISCUSSION THREAD

**Discussion by dburr** [frntc/SIDKick#5] [Q&A]
My C64 is a bit of an odd duck, apparently it's an earlier model C64C, as it has a longboard (250466) rather than the usual C64C shortboard. So I'm not sure which variant of the SIDkick I should use, the 0.3/04 or 0.3s/0.4s?

**Comment by frntc** [comment]
On the 466 the SID-socket is very close to the video connector (as for the other long boards), i.e. you need the longboard version. I recommend the 0.4, which has all the extension pins.
